SourceX/miniwin/ddraw.cpp

```diff
@@ -14,4 +15,24 @@ SDL_Surface *GetOutputSurface() {
 #endif
 }
 
+bool OutputRequiresScaling()
+{
+#ifdef USE_SDL1
+	return SCREEN_WIDTH != GetOutputSurface()->w;

```

Comment:
Width will probably be dynamically set to device width in the not to distant future, would it be good enough to check heighth or both instead?

Reply:
Done, now also checking height

Reply:
Just checking the height would not be enough, because of non-square pixels (the screen is 320x480 and we only want to downscale horizontally).